.github/workflows/ci.yml

```diff
@@ -11,18 +11,22 @@ jobs:
         timeout-minutes: 25
         env:
             JAVA_TOOL_OPTIONS: -Xmx6g
-            GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2 -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.parallel=true
+            GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2 -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy=in-process
         steps:
             - uses: actions/checkout@v2
             - uses: gradle/wrapper-validation-action@v1
             - uses: actions/setup-java@v1
               with:
                   java-version: 11
-            - uses: actions/cache@v1
+            - uses: actions/cache@v2
```

Comment:
As we are updating the cache, Github seems to suggest removing a couple of files before the cache is created:

https://docs.github.com/en/actions/guides/building-and-testing-java-with-gradle#caching-dependencies

This might resolve some issues where the cache occasionally gets stuck.

Reply:
`  - name: Cleanup Gradle Cache
    # Remove some files from the Gradle cache, so they aren't cached by GitHub Actions.
    # Restoring these files from a GitHub Actions cache might cause problems for future builds.
    run: |
      rm -f ~/.gradle/caches/modules-2/modules-2.lock
      rm -f ~/.gradle/caches/modules-2/gc.properties`

Reply:
Added this step